recipe/meta.yaml

```diff
@@ -335,15 +363,15 @@ outputs:
         <br/> Third Party Programs: Included in software package <br/><br/>
     test:
       commands:
-        - ls -A1 ${PREFIX}/lib/*        # [linux64]
+        - ls -A1 ${PREFIX}/lib/*        # [linux]
         - dir %PREFIX%\Library\lib\*    # [win64]
 
   - name: mkl-devel-dpcpp
-    script: repack.sh   # [linux64]
+    script: repack.sh   # [linux]
     script: repack.bat  # [win64]
     build:
-      number: {{ mkl_dpcpp_build_number|int + dst_build_number|int }}   # [linux64 or win64]
-      skip: True  # [not (linux64 or win64)]
+      number: {{ mkl_dpcpp_build_number|int + dst_build_number|int }}   # [linux or win64]
```

Comment:
most changes here are in that broken and/or incomplete transition of selectors ... not sure if we need to spend time on such changes.  Especially if they make things not the better

Reply:
Changed all selectors to win, osx and linux. 
Updated main build skip to # [not x86].